luaui/RmlWidgets/gui_awards/gui_awards.rml

```diff
@@ -0,0 +1,102 @@
+<rml>
+    <head>
+    <meta charset="UTF-8" />
+        <link type="text/rcss" href="/LuaUI/RmlWidgets/gui_awards/gui_awards.rcss"/>
```

Comment:
Little thing after you change file name but no need to go back up the tree, keep it clean with
```
<link type="text/rcss" href="rml_awards.rcss"/>
```